common/resources.yaml

```diff
@@ -57,9 +52,15 @@ groups:
     name: objects
     doc: A table for identifying which objects in a file contain references to external resources.
     dtype:
+    - name: files_idx
+      dtype: int64
+      doc: The row idx for the file_id in FileTable containing the object.
     - name: object_id
       dtype: text
       doc: The UUID for the object.
+    - name: object_type
+      dtype: text
+      doc: The type of the object. This is also the parent in relative_path.
```

Comment:
I see in HDMF that this change allows a convenience method `er.get_object_type`. I forget - is there another purpose to this change?

Reply:
1. The user will be able to retrieve info by searching for a type (per Ben's request)
2. The user will have information of what the parent is in the case of relative_path